ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-release-pipeline.yml

```diff
@@ -547,6 +556,16 @@ spec:
 
 
   finally:
+    - name: release-lease
+      when:
+        - input: $(tasks.status)
+          operator: in
+          values: ["Succeeded", "Completed"]
```

Comment:
What's the point of this condition? Shouldn't the lock be released no matter what's the status?

Reply:
I think yes we don't need this when condition here. I will remove this.